core/src/main/java/org/apache/accumulo/core/security/crypto/CryptoUtils.java

```diff
@@ -41,23 +41,16 @@ public static SecureRandom getSha1SecureRandom() {
     return getSecureRandom("SHA1PRNG", "SUN");
   }
 
-  public static SecureRandom getSecureRandom(String secureRNG, String secureRNGProvider) {
+  private static SecureRandom getSecureRandom(String secureRNG, String secureRNGProvider) {
```

Comment:
You could name this method "newSecureRandom" and the other "newSha1" to show that it is creating a new object every time it is called.  This kinda goes along with what I recently did in CryptoServiceFactory.